--- webfm-HEAD/views/webfm_views.inc	2009-09-02 17:10:11.000000000 +0200
+++ /webfm-HEAD/views/webfm.views.inc	2010-03-15 00:50:28.000000000 +0100
@@ -1,11 +1,11 @@
 <?php
 // $Id: webfm_views.inc,v 1.1 2009/09/02 15:10:11 robmilne Exp $
-
-
-
 function webfm_views_handlers() {
   return array(
-    'handlers' => array(
+   'info' => array(
+ 	'path' => drupal_get_path('module', 'webfm').'/views',
+	),
+   'handlers' => array(
       // Fields
       'webfm_views_handler_file_name' => array(
         'parent' => 'views_handler_field',
@@ -86,7 +86,7 @@ function webfm_views_data(){
     );
 		
 		
-		$data['webfm_file']['uid'] = array(
+  $data['webfm_file']['uid'] = array(
       'title' => t('User ID'),
       'argument' => array(
         'handler' => 'views_handler_argument_numeric',
@@ -425,4 +425,4 @@ function webfm_views_data(){
 //                                      'help' => t('Sort by file downloaded count')))
 //                                );
   return $data;
-}
\ No newline at end of file
+}
